docs/tutorials/Canopy/soil_plant_hydrology_tutorial.jl

```diff
@@ -0,0 +1,457 @@
+# # Coupling the CliMA Canopy and Soil Hydraulics Models
+
+# In the previous tutorial, we demonstrated how to run the canopy model in 
+# standalone mode using prescribed values for the inputs of soil hydraulics 
+# into the canopy hydraulics model. However, ClimaLSM has the built-in capacity 
+# to couple the canopy model with a soil physics model and timestep the two 
+# simulations together to model a canopy-soil hydrology system. This tutorial demonstrates
+# how to setup and run a coupled hydrology simulation, again using initial conditions,  
+# atmospheric and radiative flux conditions, and canopy properties observed at
+# the US-MOz flux tower, a flux tower located within an oak-hickory forest in
+# Ozark, Missouri, USA. See Wang et al. 2021
+# (https://doi.org/10.5194/gmd-14-6741-2021) for details on the site and
+# parameters.
+
+
+# In ClimaLSM, the coupling of the canopy and Richard's models is done by 
+# pairing the root extraction of the canopy hydraulics model to a ['source term'](@ref ClimaLSM.Soil.AbstractSoilSource) 
```

Comment:
"root extraction of the canopy hydraulics model, which acts as a boundary flux for the plant system, with a ['source term']..."